Makefile

```diff
@@ -24,7 +24,10 @@ host = $(shell hostname | awk '{if     (/typhon/)  print "typhon"; \
 # Need to link FFFTW against FFTW3
 # Flags which work using MacPorts on OS X:
 ifeq ($(host),other)
-    FFFTWOPTS = -I/opt/local/include -L/opt/local/lib -lfftw3 -lfftw3f
```

Comment:
Sorry, but I won't accept any changes where you've broken the default build on one system (mine) at the expense of another (yours).  I'd be really happy to see a way of setting this more generally if you want to add that!